apps/stripe/src/modules/stripe/stripe-client.ts

```diff
@@ -20,7 +20,7 @@ export class StripeClient {
       typescript: true,
       httpClient: Stripe.createFetchHttpClient(fetch), // this allow us to mock the fetch
       appInfo: {
-        name: "Saleor App Payment Stripe",
+        name: "Saleor App Payment Stripe v2",
         version: pkg.version,
         url: "https://apps.saleor.io/apps/stripe",
```

Comment:
Change this after we release Stripe v2 to app store

Reply:
I think it's fine to keep just "stripe" in app store, I don't think we need to show versioning there, for users its just stripe, there is no other one